lua/autorun/server/sv_daily_restart.lua

```diff
@@ -547,6 +557,28 @@ function CFCDailyRestart.stopSoftRestart( ply, hidePrint )
     sendAlertToClients( "The soft restart has been canceled." )
 end
 
+
+local forcedHardRestartWithConcmd = false
+
+concommand.Add( "cfc_daily_restart_force_hard_restart", function( ply )
```

Comment:
this might be nice as a ulx command so staff can use it if the server is somehow severely broke (broken physics etc) but this is probably fine for now.